tests/realtime/realtime.go

```diff
@@ -19,20 +18,33 @@ import (
 	"kubevirt.io/kubevirt/tests/console"
 	cd "kubevirt.io/kubevirt/tests/containerdisk"
 	"kubevirt.io/kubevirt/tests/framework/checks"
+	"kubevirt.io/kubevirt/tests/libvmi"
 	"kubevirt.io/kubevirt/tests/util"
 )
 
-const tuneAdminRealtimeCloudInitData = `#cloud-config
+const (
+	memory                         = "512Mi"
+	tuneAdminRealtimeCloudInitData = `#cloud-config
 password: fedora
 chpasswd: { expire: False }
 bootcmd:
    - sudo tuned-adm profile realtime
 `
-
-var (
-	memoryRequest = resource.MustParse("512Mi")
 )
 
+func newFedoraRealtime(realtimeMask string) *v1.VirtualMachineInstance {
+	return libvmi.New(
```

Comment:
Why can't we reuse `libvmi.NewFedora`?

Reply:
It is using a different containerdisk (`cd.ContainerDiskFedoraRealtime`), so it needs to be defined explicitly. The only common option is `WithRng`. Would you still favor using `libvmi.NewFedora`?

Reply:
All good.